src/platform/audio/ffmpegaudiofilecache.cpp

```diff
@@ -77,6 +79,7 @@ bool FFmpegAudioFileCache::Open_FFmpeg_Contexts(FFmpegOpenAudioFile *open_audio,
     av_log_set_level(AV_LOG_INFO);
 #endif
 
+    // REMOVE: this is required for FFmpeg older than 4.0 -> deprecated afterwards though
```

Comment:
Is there a macro for the FFMPEG version? If so, then this call can be guarded according to the version.

Reply:
Unfortunately, there's only a runtime check - it's not possible to check the version during compiletime to my knowledge

Reply:
Looks to me like there are API version macros in version.h for each library?

Reply:
Added a version check